templates/CRM/Pledge/Form/Pledge.tpl

```diff
@@ -147,10 +154,23 @@ cj(document).ready( function() {
     cj('.crm-ajax-accordion .crm-accordion-header').one('click', function() {
       loadPanes(cj(this).attr('id'));
     });
+    cj('#currency').on('change', function() {
+      replaceCurrency(cj(this).children(':selected').text());
+    });
     cj('.crm-ajax-accordion:not(.collapsed) .crm-accordion-header').each(function(index) {
       loadPanes(cj(this).attr('id'));
-      });
+    });
 });
+
+function replaceCurrency(val) {
+  var eachPaymentAmout = cj('#eachPaymentAmount');
+  var pos = val.indexOf("(") + 1;
+  if (pos) {
+    val = val.slice(pos, val.lastIndexOf(")"));
+  }
+  cj('.currency-symbol').text(val).append("&nbsp;").append(eachPaymentAmout);
+}
+
```

Comment:
I feel this is an outdated pattern.  We no longer use cj. 
The way to update this code would be to wrap the whole thing in CRM.$(function($) { ... }); and replace all cj with $.